index.html

```diff
@@ -4,6 +4,12 @@
 		<title>Minecraft Double Smooth Stone Slab</title>
 		<link rel="shortcut icon" type="image/png" href="/assets/favicon.png" />
 		<link rel="icon" type="image/png" href="/assets/favicon.png" />
+		<meta property="og:title" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:site_name" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:url" content="https://minecraftdoublesmoothstoneslab.website">
+		<meta property="og:description" content="Minecraft Double Smooth Stone Slab">
+		<meta property="og:type" content="article">
+		<meta property="og:image" content="https://minecraftdoublesmoothstoneslab.website/assets/minecraftdoublesmoothstoneslab.jpg">
```

Comment:
this asset should be at the site root so we have a dumb link
```suggestion
		<meta property="og:image" content="/minecraftdoublesmoothstoneslab.jpg">
```

Reply:
actually discuss with me if this should be in assets or not, I'm not sure if I care about having the root link

Reply:
@opl- commented that the spec always uses the full url, so probably a good idea to use it here too. I can move it out of assets, but I think it's nice to have all the images in one place

Reply:
Not having it in `/assets` allows for a really fancy URL: `https://minecraftdoublesmoothstoneslab.website/minecraftdoublesmoothstoneslab.jpg`. I'd say lets move it out.

Reply:
Moved out.